test/services/block.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+var sinon = require('sinon');
+var should = require('chai').should();
+var Promise = require('bluebird');
+
+var bitcore = require('bitcore');
+
+var BlockService = require('../../lib/services/block');
+
+describe('BlockService', function() {
+
+  it('initializes correctly', function() {
+    var database = 'database';
+    var rpc = 'rpc';
+    var txService = 'txService';
+    var blockService = new BlockService({
+      database: database,
+      rpc: 'rpc',
+      transactionService: 'txService'
+    });
+    blockService.should.exist;
```

Comment:
`should.exist(blockService);` is more correct, but I don't think it's necessary tbh